src/main/java/view/InputView.java

```diff
@@ -0,0 +1,41 @@
+package view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+
+    private static final String NAME_DELIMITER = ",";
+    private final Scanner scanner;
+
+    public InputView() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public List<String> sendNames() {
+        System.out.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
```

Comment:
사소한 부분이지만 여기도 String.format 을 사용해 DELIMITER 를 표현해보면 어떨까요?
그럴일은 없겠지만 구분자가 변경됐을 때 사용자에게 보여주는 부분에 대한 수정을 깜빡하는 휴먼에러를 방지할 수 있을것같아요 😄 

```suggestion
        System.out.println(String.format("참여할 사람 이름을 입력하세요. (이름은 쉼표(%s)로 구분하세요)", NAME_DELIMITER));
```

Reply:
위 리뷰랑 별개로 
입력에 대한 가이드를 `출력`하는것인데, OutputView가 아닌 InputView 에 넣으신 이유가 궁금해요!

Reply:
입력 - 프로그램 실행 - 출력이라는 실행 흐름의 순서로 봤을 때 입력 안내문구가 출력되는 부분은 입력의 영역이라고 생각했기 때문입니다. 

Outputview가 담당하는 출력은 프로그램의 어떠한 로직이 실행된 후 성공 또는 실패한 결과를 보여주는 것이라고 생각했습니다.